virttest/staging/utils_koji.py

```diff
@@ -1,12 +1,17 @@
-import HTMLParser
-import ConfigParser
 import os
 import time
 import logging
-import urllib
+try:
+    import configparser as ConfigParser
+except ImportError:
+    import ConfigParser
 
 from avocado.utils import path
 from avocado.utils import download
+from six.moves import html_parser
```

Comment:
HTMLParser.HTMLParser rename to html.parser on python3, please use try .. except to do import